uportal-war/src/main/java/org/jasig/portal/rest/PeopleRESTController.java

```diff
@@ -102,4 +103,18 @@ public ModelAndView getPerson(@PathVariable String username,
         return mv;
     }
 
+    @RequestMapping(value = "/people/me.json", method = RequestMethod.GET)
```

Comment:
Pulled out of pre-4-3 now. `self` is a common idiom in REST APIs for a fully-qualified URL to the current resource. In most of the popular APIs I've used, `me` is the idiom I've seen used ([reddit](https://www.reddit.com/dev/api/) for one example; probably the first place I saw this).

I've just finished looking around at a few more popular REST apis. None of them (aside from Reddit) had either `me` or `self` endpoints. Nearly all of them were OAuth-authenticated now anyway, so of course you get that information from the oauth provider's `user_info` endpoint. But since uPortal isn't an OAuth provider (yet 😄) we should probably avoid that to avoid confusion.

Just give me a holler if you want me to change it to `/self`, but my vote is for `/me` given that it's already in use and won my ad-hoc popularity search by a paltry 1-0.